src/pages/StudyBuddyPage/components/ConfirmModal/ConfirmModal.tsx

```diff
@@ -0,0 +1,28 @@
+import Button, { ButtonsEnum } from "../Button/Button";
+import style from "./ConfirmModal.module.css";
+
+export default function ConfirmModal({
+  getConfirmation,
+  text,
+}: {
+  getConfirmation: any;
```

Comment:
shouldn't be any, looks like a function that takes a boolean and returns something or returns void?

Reply:
changed it in the other pr (ticket 47, pr #64)